### DISCUSSION TITLE
I got an achievement with a repo that doesn't exist anymore.

### DISCUSSION THREAD

**Discussion by u-n-joe** [community/community#35268] [Other Feature Feedback, Questions, & Ideas] [answered]
I got an achievement with a repo that doesn't exist anymore. I don't like the fact that this achievement is untraceable and I want to get it removed from my profile because I basically can't verify I deserved this. It's more like a ghost achievement right now. I'd like to regain the achievement once I complete the requirements to get it. The 'inaccessible' below the achievement makes no sense to me.

**Comment by superbToDo** [comment, marked as answer]
ummm

**Comment by zokalinga** [comment]
Salu qui peut me débloque dans l'installation de Lollms ? 
![
![Capture2](https://github.com/user-attachments/assets/972456cf-62d7-48d7-b6cb-882fd6d5bcb5)
